pkg/controller/machineset/machineset_controller.go

```diff
@@ -175,8 +175,28 @@ func (c *MachineSetController) addMachineSet(obj interface{}) {
 
 func (c *MachineSetController) updateMachineSet(old, cur interface{}) {
 	ms := cur.(*clusteroperator.MachineSet)
+	oldms := cur.(*clusteroperator.MachineSet)
 	loggerForMachineSet(c.logger, ms).Debugf("enqueueing updated machine set")
 	c.enqueueMachineSet(ms)
+	if !oldms.Status.Installed && ms.Status.Installed {
```

Comment:
Should this be done only for when the machine set is a master? Or maybe a master or an infra?